docs/examples/vllm/TPU/README.md

```diff
@@ -0,0 +1,118 @@
+# Deploy Distributed Inference Service with vLLM and LWS on TPUs
+
+In this example, we will use LeaderWorkerSet to deploy a distributed inference service with vLLM on TPUs. It manages the distributed runtime with [Ray](https://docs.ray.io/en/latest/index.html).
+
+## Install LeaderWorkerSet
+
+Follow the step-by-step guide on how to install LWS. [View installation guide](https://github.com/kubernetes-sigs/lws/blob/main/docs/setup/install.md)
+
+
+## Deploy LeaderWorkerSet of vLLM
+We use LeaderWorkerSet to deploy two vLLM model replicas, and each vLLM replica has 4 pods, and 4 TPUs per pod (tensor_parallel_size=16). 
```

Comment:
```suggestion
In this example, we use LeaderWorkerSet to deploy two vLLM server replicas to serve the llama3-70b model on two v5e-16 TPU slices. Since the v5e-16 TPU slice has four hosts, so each vLLM replica will have 4 workers, and each worker will consume 4 TPUs (with tensor_parallel_size=16). 
```